platform/x11/detect.py

```diff
@@ -157,12 +157,13 @@ def configure(env):
     env.ParseConfig('pkg-config xrandr --cflags --libs')
     env.ParseConfig('pkg-config xrender --cflags --libs')
 
+    x11_error = os.system("pkg-config xi --modversion > /dev/null ")
```

Comment:
Please move it with other checks in `can_build` (and there no need to call `sys.exit(255)`.